src/main/java/com/opentok/PatchProperties.java

```diff
@@ -0,0 +1,115 @@
+/**
+ * OpenTok Java SDK
+ * Copyright (C) 2021 Vonage.
+ * http://www.tokbox.com
+ *
+ * Licensed under The MIT License (MIT). See LICENSE file for more information.
+ */
+package com.opentok;
+
+/**
+ * Defines values for the <code>properties</code> parameter of the patch archive and broadcast
+ * Internal only
+ */
+public class PatchProperties {
```

Comment:
This class should not be public. Same for PatchProperties.Builder. (I noticed this while generating the JavaDoc content.) Can we make it internal (or protected) or something that will not show up in the docs?

Reply:
`PatchProperties` are used in `httpClient` so I can not make them package local.

Reply:
all the other properties are also public.